README.pod

```diff
@@ -98,40 +96,171 @@ C<dispose> method:
 
   $dbh.dispose;
 
-=head2 Fetching data
+=head2 Running Queries
+
+=head3 C<execute>
+
+For a single execution of a query you may use execute directly. This starts the query within the database and returns
+a C<StatementHandle> which will be used for [Retrieving Data](#retrieving-data).
```

Comment:
`can be used` because queries like the example right below don't return any data.